llmfoundry/command_utils/train.py

```diff
@@ -502,6 +519,18 @@ def train(cfg: DictConfig) -> Trainer:
 
     _log_num_params(model, logged_cfg)
 
+    # TP config
+    if tp_config is not None:
+        if 'layer_plan' not in tp_config:
+            tp_config['layer_plan'] = {}
+        if 'strategy' in tp_config:
+            strategy = tp_config.pop('strategy')
+            strategy_layer_plan = build_tp_strategy(strategy, model)
+            tp_config['layer_plan'] |= strategy_layer_plan
```

Comment:
why is this a |=?

Reply:
Removed.

Originally we thought the user was going to provide a layer plan and we'd want to combine that with the layer plan of the strategy but then we decided to not support this.